agent/consul/leader_peering_test.go

```diff
@@ -336,11 +332,6 @@ func insertTestPeeringData(t *testing.T, store *state.Store, peer string, lastId
 				Node:        "bbb",
 				PeerName:    peer,
 			},
-			{
-				CheckID:  structs.SerfCheckID,
```

Comment:
these checks will actually cause reaping of the nodes -- let's not use them at all